### DISCUSSION TITLE
Lesson 6: EIP -385 ins not supportedin one or more of the RPCs used

### DISCUSSION THREAD

**Discussion by dezlymacauley** [Cyfrin/foundry-full-course-cu#391] [Q&A]
![2](https://github.com/Cyfrin/foundry-full-course-f23/assets/120219951/6c7ee8bd-43b5-435e-9159-369cae0ef341)

I'm getting this warning when I try to run my DeploySimpleStorage script in anvil after using the following commands

```bash
anvil
```

Then on a new terminal

```bash
forge script script/DeploySimpleStorage.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --private-key xxxxxx...
```
For some context both my DeploySimpleStorage.s.sol and SimpleStorage.sol are both using solidity 0.8.19
The transaction does deploy and I get the "ONCHAIN EXECUTION COMPLETE & SUCCESSFUL" message

**Comment by garyghayrat** [comment]
Hi, could you try running `foundryup` then restarting your terminal, anvil node, and then try running the script again?

**Reply by dezlymacauley** [reply]
It's still giving me the same error. But It's all good. Managed to deploy to Third Web.
Hopefully this message doesn't come back to bite me in the butt later

**Reply by garyghayrat** [reply]
Sounds good, I'll keep an eye out for solutions. Good luck.

**Reply by dezlymacauley** [reply]
@garyghayrat Best of luck to you as well :)
